pkg/tfbridge/provider.go

```diff
@@ -1585,8 +1590,7 @@ func (p *ProviderInfo) SetAutonaming(maxLength int, separator string) {
 				sch.Type() == shim.TypeString { // has type string
 
 				if _, hasfield := res.Fields[nameProperty]; !hasfield {
-					ensureMap(&res.Fields)[nameProperty] =
-						AutoName(nameProperty, maxLength, separator)
+					ensureMap(&res.Fields)[nameProperty] = AutoName(nameProperty, maxLength, separator)
```

Comment:
This is not a meaningful change right? We should agree on column width length (or none). Maybe gowumpt is strict enough to just make this happen?

Reply:
I think this is because I have go 1.21 locally and the bridge uses 1.20 in CI. I have enabled gofmt to run on save and I get these spurious diffs in the bridge because of that.